src/components/App/index.js

```diff
@@ -143,6 +144,7 @@ export class App extends Component {
 		await loadConfig();
 		this.handleTriggers();
 		CustomFields.init();
+		Language.init();
```

Comment:
Why did you create a `class` to handle Language stuff instead of just exporting the methods you need to use?